content/docs/1.6.0/deploy/install/install-with-argocd.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Install with ArgoCD
+weight: 12
+---
+
+## Prerequisites
+- Your workstation: Install [ArgoCD CLI](https://argo-cd.readthedocs.io/en/stable/cli_installation/).
+- Kubernetes cluster:
+  - Ensure that each node fulfills the [installation requirements](../#installation-requirements).
+  - Install [ArgoCD](https://argo-cd.readthedocs.io/en/stable/).
+
+    ```bash
+    kubectl create namespace argocd
+    kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/core-install.yaml
+    ```
+    Allow some time for the deployment of ArgoCD components in the `argocd` namespace.
+
+> Use [this script](https://github.com/longhorn/longhorn/blob/v{{< current-version >}}/scripts/environment_check.sh) to check the Longhorn environment for potential issues.
+
+## Installing Longhorn
+
+1. Log in to ArgoCD
+
+    ```bash
+    argocd login --core
+    ```
+
+2. Set the current namespace to `argocd`:
+
+    ```bash
+    kubectl config set-context --current --namespace=argocd
+    ```
+
+3. Create the Longhorn Application custom resource.
+
+    ```bash
+    cat > longhorn-application.yaml <<EOF
+    apiVersion: argoproj.io/v1alpha1
+    kind: Application
+    metadata:
+      name: longhorn
+      namespace: argocd
+    spec:
+      project: default
+      sources:
+        - chart: longhorn
+          repoURL: https://charts.longhorn.io/
+          targetRevision: v1.6.0 # Replace with the Longhorn version you'd like to install or upgrade to
+          helm:
+            values: |
+              helmPreUpgradeCheckerJob:
+                enabled: false
+      destination:
+        server: https://kubernetes.default.svc
+        namespace: longhorn-system
+    EOF
+    kubectl apply -f longhorn-application.yaml
+    ```
+
+4. Sync (Deploy) the Application.
+
+    ```bash
+    argocd app sync longhorn
+    ```
+
+    This command will synchronize the Longhorn Application, deploying Longhorn with the specified configurations.
```

Comment:
Please remove this. If users want a more general description of the `argocd app sync` command, they can read the Argo CD documentation.